tests/encryption_/test_management.py

```diff
@@ -0,0 +1,148 @@
+                "queries": {"queryType": "range"},
+            },
+            {
+                "bsonType": "binData",
+                "path": "profile_picture",
+                "queries": {"queryType": "equality"},
+            },
+            {
+                "bsonType": "int",
+                "path": "patient_age",
+                "queries": {"queryType": "range"},
+            },
+            {
+                "bsonType": "double",
+                "path": "weight",
+                "queries": {"queryType": "range"},
+            },
+        ]
+    },
+    "encryption__patient": {
+        "fields": [
+            {
+                "bsonType": "int",
+                "path": "patient_id",
+                "queries": {"queryType": "equality"},
+            },
+            {
+                "bsonType": "string",
+                "path": "patient_name",
+            },
+            {
+                "bsonType": "string",
+                "path": "patient_notes",
+                "queries": {"queryType": "equality"},
+            },
+            {
+                "bsonType": "date",
+                "path": "registration_date",
+                "queries": {"queryType": "equality"},
+            },
+            {
+                "bsonType": "bool",
+                "path": "is_active",
+                "queries": {"queryType": "equality"},
+            },
+            {"bsonType": "string", "path": "email", "queries": {"queryType": "equality"}},
+        ]
+    },
+    "encryption__patientportaluser": {
+        "fields": [
+            {"bsonType": "string", "path": "ip_address", "queries": {"queryType": "equality"}},
+            {"bsonType": "string", "path": "url", "queries": {"queryType": "equality"}},
+        ]
+    },
+    "encryption__encryptednumbers": {
+        "fields": [
+            {"bsonType": "int", "path": "pos_bigint", "queries": {"queryType": "equality"}},
+            {"bsonType": "int", "path": "pos_smallint", "queries": {"queryType": "equality"}},
+            {"bsonType": "int", "path": "smallint", "queries": {"queryType": "equality"}},
+        ]
+    },
+    "encryption__appointment": {
+        "fields": [{"bsonType": "date", "path": "time", "queries": {"queryType": "equality"}}]
+    },
+}
+
+
+@modify_settings(
+    INSTALLED_APPS={"prepend": "django_mongodb_backend"},
+)
+@override_settings(DATABASE_ROUTERS=[TestEncryptedRouter()])
+class EncryptedFieldsManagementCommandTests(TransactionTestCase):
+    databases = {"default", "encrypted"}
+    available_apps = ["django_mongodb_backend", "encryption_"]
+
+    def _compare_json(self, json1, json2):
+        # Remove keyIds since they are different for each run.
+        for table in json2:
+            for field in json2[table]["fields"]:
+                del field["keyId"]
+        # TODO: probably we don't need to test the entire mapping, otherwise it
+        # requires updates every time a new model or field is added!
```

Comment:
Can this be an INTPYTHON task if it's not in scope for this PR? It'd be a good starter tasks for newer memberos f the team.

Reply:
done